spec/unit/models/attribute_spec.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require "sequel"
+
+module RightsAPI
+  RSpec.describe(Attribute) do
+    let(:hash_keys) { %i[id type name description].sort.freeze }
+
+    describe "#base_dataset" do
+      it "returns self" do
+        expect(described_class.base_dataset).to eq(described_class)
+      end
+    end
+
+    describe "#default_key" do
+      it "returns a Symbol" do
+        expect(described_class.default_key).to be_an_instance_of(Symbol)
+      end
+    end
+
+    describe "#default_order" do
+      it "returns SQL expression" do
+        expect(described_class.default_order).to be_a(Sequel::SQL::QualifiedIdentifier)
+      end
+    end
+
+    describe "#to_h" do
+      it "returns expected structure" do
+        expect(described_class.first.to_h.keys.sort).to eq(hash_keys)
```

Comment:
In these test classes, you keep declaring `hash_keys` with `let`, and then using the variable exactly once. My understanding of `let` is that it makes sense use when declaring variables that you intend to use in several tests and get it reset in between tests. Since that is not how you are using `let`, I'd just declare `hash_keys` as a regular variable inside the test that uses it, in this case `"returns expected structure"`.

Reply:
Done